src/src.pro

```diff
@@ -17,6 +17,17 @@ qmldir.files += $$_PRO_FILE_PWD_/qmldir $$_PRO_FILE_PWD_/pyotherside.qmltypes
 qmldir.path += $$target.path
 INSTALLS += qmldir
 
+qmlplugindump.target = plugins.qmltypes
+qmlplugindump.depends = target
+qmlplugindump.commands = qmlplugindump io.thp.pyotherside 1.4 . > plugins.qmltypes
```

Comment:
Please read the version number by including `pyotherside.pri` and then using the `VERSION` variable. If we need a special plugin version variable, we can add that to `pyotherside.pri` (e.g. `PLUGIN_VERSION`).